examples/ec2-rds-with-vpc.nix

```diff
@@ -0,0 +1,71 @@
+let
+  region = "us-east-1";
+  accessKeyId = "AKIA...";
+in
+  {
+    network.description = "NixOps RDS in a VPC Testing";
+  # A VPC.
+  resources.vpc.private = {
```

Comment:
Can you also fix the indentation in this file please and remove any comments such as `# A VPC` which don't seem to be useful.